case-lib/config.sh

```diff
@@ -34,6 +34,10 @@ LOG_ROOT=${LOG_ROOT:-}
 # EXAMPLE: the sof-logger will collect the data into this folder
 SOFLOGGER=${SOFLOGGER:-}
 
+# User may not put sof-*.ldc into the default folder "/etc/sof", this env allows
+# user to specify the path of their own sof-*.ldc file.
+SOFLDC=${SOFLDC:-}
```

Comment:
I don't understand the purpose of these lines. The only thing they do is changing a missing variable to an empty variable. As we don't use `set -o nounset` this doesn't seem to make any difference.

Reply:
I didn't realize this before, if you say so, I think most content of config.sh can be dropped, but for now, can we just align with  others, eg: LOG_ROOT, SOFLOGGER, etc? I can drop all of them in a new PR.

Reply:
If I install something in the default location and the scripts find it in the default location then everything just works and I don't want to hear about implementation details, it's just noise.

If I forget to install something or put in the wrong location, _then_ I want the error message or `env-check.sh` to help me fix it.

Either way I don't think such a list is useful. I didn't know about this list until a couple days ago and I never missed it.

A list of _required_ options that don't have defaults all gathered in a single place would be useful but fortunately we require almost nothing so we still don't need such a list.

This is a bit off-topic, for now in this PR I recommend to leave `config.sh` as it is.